tests/console/yast2_tftp.pm

```diff
@@ -35,14 +35,18 @@ sub run() {
     send_key 'alt-t';
     for (1 .. 20) { send_key 'backspace'; }
     type_string '/srv/tftpboot/new_dir';
-    sleep 1;
+    wait_still_screen 1;
     # open port in firewall, if firewall is disabled, then continue with next test view log
     if (check_screen 'yast2_tftp_open_port') { send_key 'alt-f'; send_key 'alt-i'; assert_screen 'yast2_tftp_firewall_details'; send_key 'alt-o'; }
 
     # view log
+    wait_still_screen;
```

Comment:
default timeout is 7 seconds which is a bit long. Can a shorter timeout be specified as argument?